Exercise_1.py

```diff
@@ -1,22 +1,38 @@
 class myStack:
      def __init__(self):
-         
+         self.stack = []
+
      def isEmpty(self):
-         
+
+     	 if self.stack is None:
```

Comment:
This will never return True, because stack is initialized as an array and will never be a None pointer.